File: Creating nodes/linked-list.py
class Node:
    def __init__(self, value):
        self.data = value
        self.next = None

node1 = Node(10)
node2 = Node(20)
node3 = Node(30)
print(f"Node1 Data: {node1.data}, Node1 next: {node1.next}")
print(f"Node2 Data: {node2.data}, Node2 next: {node2.next}")
print(f"Node3 Data: {node3.data}, Node3 next: {node3.next}")

File: Creating nodes/linked-list.cpp
#include <iostream>
using namespace std;

class Node {
public:
    int data;
    Node* next;

    Node(int value) {
        data = value;
        next = nullptr;  // No connection yet
    }
};

int main() {
    Node* node1 = new Node(10);
    cout << "Node data: " << node1->data << ", Next node: " << node1->next << endl;
    return 0;
}
